### ISSUE TITLE
PEP 3105 Make print a function in twisted/words

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8395]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8395|
|Type|enhancement|
|Created|2016-06-03 10:46:50Z|

This ticket was split out of [ticket:5812] which was too big to review.

PEP 3105 ( ​​​https://www.python.org/dev/peps/pep-3105/ ) **print** deprecates has changed from a statement to a function.

In Python 2.6 and Python 2.7, the default is print as a statement

```
print "Hello"
```

In Python 3.x, the above example is an error, since only print as a function is available.

To consistently use **print** as a function
on Python 2.6, Python 2.7, and Python 3.x, this should be done:

```
from __future__ import print_function

print("Hello")
```

<details><summary>Searchable metadata</summary>

```
trac-id__8395 8395
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__words words
keywords__None None
time__1464950810324029 1464950810324029
changetime__1465029897227854 1465029897227854
version__None None
owner__Ying_Li__cyli_____ Ying Li <cyli@...>
cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/111

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_5"></a>|@cyli set owner to @cyli<br>@cyli set status to `closed`|
|-|-|

In changeset 592a4f992eff94aac1518cf06074ad22e10ecdee
```
#!CommitTicketReference repository="" revision="592a4f992eff94aac1518cf06074ad22e10ecdee"
Merge pull request #111 from rodrigc/8395-rodrigc-print

Author: rodrigc
Reviewer: cyli
Fixes: #8395

Convert from print statement to print() function in twisted/words
```
